src/arkode/arkode_mristep.c

```diff
@@ -3571,6 +3571,11 @@ int mriStep_StageERKFast(ARKodeMem ark_mem, ARKodeMRIStepMem step_mem,
                         __FILE__, "Unable to get accumulated error from the inner stepper");
         return (ARK_INNERSTEP_FAIL);
       }
+
+      /* scale the error estimate by 1/rtol to account for different inner/outer tolerances */
+      step_mem->inner_dsm /= ark_mem->reltol;
+      SUNLogInfo(ARK_LOGGER, "end-fast-steps", "inner_dsm = %e",
+                 step_mem->inner_dsm);
```

Comment:
`end-fast-steps` is a special key the parser uses to separate the multirate levels so this needs to use a different key
```suggestion
      SUNLogInfo(ARK_LOGGER, "accumulated-fast-error", "inner_dsm = %e",
                 step_mem->inner_dsm);
```